dist/WebVtt.js

```diff
@@ -1,4 +1,7 @@
-export class VttCueSettings {
+"use strict";
```

Comment:
maybe gitignore the dist folder altogether?

Reply:
i'm not sure whether that is a good idea, since the dist folder is that is being uploaded to npm and that we there is atleast some proof that we're uploading the right code. But we can do so if you wish

Reply:
I think i would prefer that. We could still cheat and commit unrelated dist code to this repo

Reply:
and maybe switch to github actions?